logging/cluster-logging-external.adoc

```diff
@@ -41,7 +41,133 @@ include::modules/cluster-logging-collector-log-forward-syslog.adoc[leveloffset=+
 
 include::modules/cluster-logging-collector-log-forward-cloudwatch.adoc[leveloffset=+1]
 
+=== Forwarding logs to Amazon CloudWatch from STS enabled clusters
+
+For clusters with AWS Security Token Service (STS) enabled, you can create an AWS service account manually or create a credentials request using the xref:../authentication/managing_cloud_provider_credentials/about-cloud-credential-operator.adoc[Cloud Credential Operator(CCO)] utility `ccoctl`.
+
+.Creating an AWS credentials request
+. Create a `CredentialsRequest` Custom Resource YAML using the template below:
++
+.CloudWatch Credentials Request Template
+[source,yaml]
+----
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  name: <your_role_name>-credrequest
+  namespace: openshift-logging
+spec:
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: AWSProviderSpec
+    statementEntries:
+      - action:
+          - logs:PutLogEvents
+          - logs:CreateLogGroup
+          - logs:PutRetentionPolicy
+          - logs:CreateLogStream
+          - logs:DescribeLogGroups
+          - logs:DescribeLogStreams
+        effect: Allow
+        resource: arn:aws:logs:*:*:*
+  secretRef:
+    name: <your_role_name>
+    namespace: openshift-logging
+  serviceAccountNames:
+    - logcollector
+----
++
+. Use the `ccoctl` command to to create a role for AWS using your `CredentialsRequest` CR. With the `CredentialsRequest` object, this `ccoctl` command creates an IAM role with a trust policy that is tied to the specified OIDC identity provider, and a permissions policy that grants permissions to perform operations on CloudWatch resources. This command also creates a YAML configuration file in ``/<path_to_ccoctl_output_dir>/manifests/openshift-logging-<your_role_name>-credentials.yaml`. This secret file contains the `role_arn` key/value used during authentication with the AWS IAM identity provider.
++
+[source,terminal]
+----
+ccoctl aws create-iam-roles \
```

Comment:
To run this command, the AWS credential must be provided. Either credential file [1] or credential 
 ENVs[2].
 [1]https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html
 [2]
```
  export AWS_ACCESS_KEY_ID=IDExample
export AWS_SECRET_ACCESS_KEY=KEYExample
export AWS_REGION=us-east-2
```

Reply:
The credential is requested in the prior step.

Reply:
@libander  I can not find the credential?  Could point me the link?

Reply:
The step immediately before this requests a credential from AWS using the ccoctl tool. 
`. Use the `ccoctl` command to to create a role for AWS using your `CredentialsRequest` CR. With the `CredentialsRequest` object, this `ccoctl` command creates an IAM role with a trust policy that is tied to the specified OIDC identity provider, and a permissions policy that grants permissions to perform operations on CloudWatch resources. This command also creates a YAML configuration file in ``/<path_to_ccoctl_output_dir>/manifests/openshift-logging-<your_role_name>-credentials.yaml`. This secret file contains the `role_arn` key/value used during authentication with the AWS IAM identity provider.`

Reply:
@libander  To be simple. We add a link this page https://docs.openshift.com/container-platform/4.10/authentication/managing_cloud_provider_credentials/cco-mode-sts.html.

Reply:
@anpingli - As an additional resource?

Reply:
Yes.

Reply:
This PR already links to https://docs.openshift.com/container-platform/4.10/authentication/managing_cloud_provider_credentials/about-cloud-credential-operator.html -- which is an overview of the tool, and contains a link to https://docs.openshift.com/container-platform/4.10/authentication/managing_cloud_provider_credentials/cco-mode-sts.html